diff -up cutmp3-2.0.2/main.c.filename_default_prefix cutmp3-2.0.2/main.c
--- cutmp3-2.0.2/main.c.filename_default_prefix	2010-07-05 21:31:38.000000000 +0200
+++ cutmp3-2.0.2/main.c	2010-07-05 21:31:46.000000000 +0200
@@ -2911,7 +2911,7 @@ int main(int argc, char *argv[])
 		}
 	}
 
-	if (strlen(prefix)==0) prefix="result";
+	if (strlen(prefix)==0) prefix=filename;
 
 	if (userin==0 && userout!=0) usage("ERROR: missing inpoint");
 	if (userin!=0 && userout==0) usage("ERROR: missing outpoint");
